src/operator/controllers/validating_webhook_configurations_controller.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"context"
+	"github.com/otterize/intents-operator/src/shared/filters"
+	"github.com/samber/lo"
+	"github.com/sirupsen/logrus"
+	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+)
+
+// ValidatingWebhookConfigsReconciler reconciles webhook configurations
+type ValidatingWebhookConfigsReconciler struct {
+	client.Client
+	Scheme  *runtime.Scheme
+	certPem []byte
```

Comment:
```suggestion
	certPEM []byte
```

Reply:
See the "Initialisms" section here: https://go.dev/wiki/CodeReviewComments#:~:text=return%0A%7D%0A//%20use%20x-,Initialisms,-Words%20in%20names